lib/taskana-core/src/test/java/acceptance/monitoring/ProvideDetailedClassificationReportAccTest.java

```diff
@@ -68,50 +65,58 @@ public static void resetDb() throws SQLException, IOException {
 
     @WithAccessId(userName = "monitor_user_1")
     @Test
-    public void testGetTotalNumbersOfTasksOfClassificationReport()
+    public void testGetTotalNumbersOfTasksOfDetailedClassificationReport()
         throws WorkbasketNotFoundException, NotAuthorizedException, ClassificationNotFoundException {
 
         TaskMonitorService taskMonitorService = taskanaEngine.getTaskMonitorService();
 
-        List<Workbasket> workbaskets = getListOfWorkbaskets();
+        List<String> workbasketIds = Arrays.asList(
+            "WBI:000000000000000000000000000000000001",

```

Comment:
Optonal: You´re generating Lists/Single Objects with ID multiple time on few test-methods. Maybe you can create a method wich does generate X-IDs, starting at Y.          Array[]/List<Ids> ids = generateIds(3, 1);

Only in cases where you got only on ID you need to use get(0)/[0] or keep them like they are. Maybe method will very usefull if tests get´s lager.